src/data/texts/smiths/names.json

```diff
@@ -0,0 +1,11 @@
+{
+  "names": [
```

Comment:
Seems to me that this should be an enum instead of a pure json.
Remember that .json files are very bad when coding TS

Reply:
Ok. I found strange to put an enum here, but I guess I have no real reason...